src/main/java/graph/Node.java

```diff
@@ -64,17 +66,25 @@ public int getColumn() {
      * Sets the column of the node.
      * @param newColumn the newColumn value of the node
      */
-    public void setColumn(int newColumn) {
+    private void setColumn(int newColumn) {
```

Comment:
Maybe this will make testing harder. We should pay attention to that when we merge with the branch Testing1,2